lib/utils.coffee

```diff
@@ -16,20 +16,53 @@ limitations under the License.
 
 isString = require('lodash/isString')
 isArray = require('lodash/isArray')
+assign = require('lodash/assign')
+
+getBaseChannel = (device) ->
+	device.logs_channel or device.uuid
+
+###*
+# @summary Get logs channel name for the given device
+# @function
+# @protected
+#
+# @param {Object} device - device
+# @returns {String} logs channel name
+#
+# @example
+# channel = utils.getChannel('...')
+###
+exports.getChannel = (device, suffix = 'logs') ->
+	return "device-#{getBaseChannel(device)}-#{suffix}"
 
 ###*
-# @summary Get logs channel name from a uuid
+# @summary Get logs channel name for the given device
 # @function
 # @protected
 #
 # @param {Object} device - device
-# @returns {String} logs channel
+# @returns {String} logs channel name
 #
 # @example
 # channel = utils.getChannel('...')
 ###
-exports.getChannel = (device) ->
-	return "device-#{device.logs_channel or device.uuid}-logs"
+exports.getChannel = (device, suffix = 'logs') ->
+	return "device-#{getBaseChannel(device)}-#{suffix}"
```

Comment:
Oops wrong. What I've noticed was the duplicate `getBaseChannel`. Looks like I had some buggy local merge.